packages/codepack/package.json

```
@@ -12,11 +12,13 @@
     "code:all": "pnpm code:01-extract && pnpm code:02-embed && pnpm code:03-cluster && pnpm code:04-name && pnpm code:05-materialize"
   },
   "dependencies": {
+    "@promethean/utils": "workspace:*",
```

Biome (2.1.2)

[error] The key @promethean/utils was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)